### PR TITLE
add missing keda namespaceLabels

### PR DESCRIPTION
**Pull request by andriisoldatenko**
based on slack conversation:
```
workers:
  autoscaling:
    enabled: false
    maxReplicas: 10
    minReplicas: 1
    targetCPUUtilization: 80
    targetMemoryUtilization: 80
  keda:
    cooldownPeriod: 30
    enabled: true
    namespaceLabels:
      name: astronomer # <-- add this line
    pollingInterval: 5
```
for ref https://github.com/astronomer/issues/issues/1625

### CODE REVIEW

File: charts/astronomer/templates/houston/api/houston-configmap.yaml [hunk 1]
```diff
@@ -207,6 +207,8 @@ data:
           # This is here for upgrading to 0.11.1+ of Airflow.
           keda:
             enabled: false
+            namespaceLabels:
+              name: {{ .Release.Name }}
```
Comment (schnie) [inline charts/astronomer/templates/houston/api/houston-configmap.yaml:211]: Just to be sure, we want Release.Name (as you have) for this and not Release.Namespace right?

Comment (andriisoldatenko) [reply charts/astronomer/templates/houston/api/houston-configmap.yaml:211]: good point... @dimberman wdyt?

**Comment by dimberman** [comment]
@schnie @andriisoldatenko it should be whatever the label is on the astronomer namespace. If I recall that might be somewhere in the terraform as the helm chart doesn't create the namespace.

File: charts/astronomer/templates/houston/api/houston-configmap.yaml (hunk 1 above)
Comment (danielhoherd) [reply charts/astronomer/templates/houston/api/houston-configmap.yaml:211]: Houston does it exactly like this. https://github.com/astronomer/astronomer/blob/master/charts/astronomer/templates/houston/api/houston-configmap.yaml#L176-L177

**Comment by schnie** [comment]
Feels like we should document this behavior and configuration somewhere. If someone doesn't use the terraform, they'll have to take an additional step to add the namespace label right?

File: charts/astronomer/templates/houston/api/houston-configmap.yaml (hunk 1 above)
Comment (schnie) [reply charts/astronomer/templates/houston/api/houston-configmap.yaml:211]: Yep @danielhoherd that link represents labels that we want to apply to every airflow namespace. I believe the new usage here under KEDA is to tell the `pgbouncer` `NetworkPolicy` to allow traffic from the KEDA controller inside the platform namespace. It needs to run a query on the deployment dbs, so it connects to these to do so.

**Comment by andriisoldatenko** [comment]
@schnie not sure that i fully understand what’s need to be done from my side…
not sure where and what document and how to check terraform part…

**Comment by schnie** [comment]
I think we need to document this part on our installation docs - 


```
$ kubectl label namespace astronomer name=astronomer
namespace/default labeled
$ kubectl describe namespace astronomer
Name:         default
Labels:       name=astronomer
Annotations:  <none>
Status:       Active

No resource quota.

No LimitRange resource.
```

@andriisoldatenko is this what a user would need to do to use this? Looks like this label was somehow applied to dev.

**Comment by schnie** [comment]
I updated the notion doc with this step https://www.notion.so/astronomerio/Using-KEDA-on-Astronomer-e15473b165d14443b17311def6979924

**Comment by schnie** [comment]
Does anyone know how this label was added to dev? I can't find it in our terraform, so I'm wondering if it was manually applied.

If it doesn't exist, we should automate this step in our terraform.

**Comment by jpweber** [comment]
It might have just been added manually via kubectl  "for testing".

**Comment by vishwas-astro** [comment]
@jpweber before we forget about it, we need to make sure to add all the necessary things for KEDA support into all our terraforms. Do you mind creating an issue for that please?

**Comment by schnie** [comment]
Created the issue over here https://github.com/astronomer/issues/issues/1654

**Comment by vishwas-astro** [comment]
Never mind. @schnie already created it.

**Pull request merged**